content/2_Bronze/Intro_Sets.mdx

```diff
@@ -1226,4 +1226,222 @@ make their implementation easier.
 
 </Quiz>
 </JavaSection>
+
+
+<PySection>
+
+<Quiz>
+	<Quiz.Question>
+		What is the time complexity of insertions, deletions, and searches in a sorted set of size $N$?
+		<Quiz.Answer>
+			$\mathcal{O}(1)$
+		</Quiz.Answer>
+		<Quiz.Answer>
+			$\mathcal{O}(N \log N)$
+		</Quiz.Answer>
+		<Quiz.Answer>
+			$\mathcal{O}(N)$
+		</Quiz.Answer>
+		<Quiz.Answer correct>
+			$\mathcal{O}(\log N)$
+			<Quiz.Explanation>
+				Correct.
+			</Quiz.Explanation>
+		</Quiz.Answer>
+	</Quiz.Question>
+	<Quiz.Question>
+		Which of the following is the correct output of this code?
+		```py
+		s = set()
+		s.add(1)
+		s.add(4)
+		s.add(5)
+		s.add(2)
```

Comment:
why not just `s = {1, 4, 5, 2}`

Reply:
similar reason to: https://github.com/cpinitiative/usaco-guide/pull/3677#discussion_r1199823416